### PR TITLE
Add micro:bit

### PR DESCRIPTION
**Pull request by carlosperate**
**https://github.com/carlosperate/awesome-microbit**

**The micro:bit is a small programmable embedded board designed to make learning and teaching easy and fun (http://microbit.org).**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- [x] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- [x] It's the result of hard work and the best I could possibly produce.
	- [x] Non-generated Markdown file in a GitHub repo.
	- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- [x] Not a duplicate.
	- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
	- [x] Includes a project logo/illustration whenever possible.
		- [x] Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- [x] The image should link to the project website or any relevant website.
		- [ ] The image should be high-DPI. Set it to maximum half the width of the original image.
	- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- [x] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- [x] Should be placed on the right side of the readme heading.
		- [x] Should link back to this list.
	- [x] Has a Table of Contents section.
		- [x] Should be named `Contents`, not `Table of Contents`.
		- [x] Should be the first section in the list.
		- [x] Should only have one level of sub-lists, preferably none.
	- [x] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- [x] The file should be named `contributing.md`. Casing is up to you.
	- [x] Has consistent formatting and proper spelling/grammar.
		- [x] The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- [x] The description starts with an uppercase character and ends with a period.
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -98,6 +98,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [micro:bit](https://github.com/carlosperate/awesome-microbit) - Tiny programmable computer, designed to make learning and teaching easy and fun.
```
Comment (carlosperate) [inline readme.md]: Description was taken for the official website, but don't know if you'd prefer something a bit different. Perhaps "computer" is not the best word here, "embedded computer" or "embedded board"?

**Comment by carlosperate** [comment]
The image currently issued in the list is perhaps not HDPI enough (and doesn't quite the entire full width of the document), but it's the best image I was able to find for it. Would that still be okay?

I believe I have covered all the other points, but of course please let me know if I have missed anything or if you have any suggestions.

File: readme.md (hunk 1 above)
Comment (carlosperate) [inline readme.md]: Oh, and micro:bit is meant to be all lower case (I know...), which is why I haven't capitalised it.

**Comment by carlosperate** [comment]
Any feedback on the couple of questions I had to complete the submission would be greatly appreciated.

**Comment by legodude17** [comment]
I don't think you need to write MicroPython at the beginning of everything in the MicroPython Libraries section.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/carlosperate/awesome-microbit
✖ Linting

  README.md:466:3
  ✖    1:1    Missing License section                                               remark-lint:awesome/license
  ✖    1:1    Missing or invalid Table of Contents                                  remark-lint:awesome/toc
  ✖   16:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   23:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   24:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   25:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   26:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   27:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   28:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   29:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   30:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   31:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   32:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   33:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   34:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   35:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   36:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   37:3    Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖  326:104  List item description must start with valid casing                    remark-lint:awesome/list-item
  ✖  413:3    Link to http://www.dfrobot.com/category-microbit.html is dead         remark-lint:no-dead-urls
  ✖  466:3    Link to http://microbit.org/en/2018-01-19-train_the_trainer/ is dead  remark-lint:no-dead-urls

  21 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**